compliance_checker/ioos.py

```diff
@@ -5,8 +5,10 @@
 from compliance_checker.base import BaseCheck, BaseNCCheck, BaseSOSGCCheck, BaseSOSDSCheck, check_has, Result
 from owslib.namespaces import Namespaces
 from lxml.etree import XPath
-from compliance_checker.cfutil import get_geophysical_variables
-from compliance_checker.cf.cf import CF1_6Check
+from compliance_checker.acdd import ACDD1_3Check
+from compliance_checker.cfutil import get_geophysical_variables, get_instrument_variables
+from compliance_checker.cf.cf import CF1_6Check, CF1_7Check
+from rfc3986 import api, exceptions, validators
```

Comment:
Probably want to replace this with `validators` package since we mainly need to check URLs, as opposed to generically checking URIs.